tests/beaker_tests/cisco_bgp_neighbor_af/test_bgpneighboraf.rb

```diff
@@ -376,11 +394,11 @@
 
 tests['non_default_properties_ebgp_only'] = {
   :desc           => "2.9 Non Default Properties: 'ebgp' commands",
-  :title_pattern  => '2 yellow 3.3.3.3 ipv4 unicast',
-  :remote_as      => '2 yellow  3.3.3.3 3',
-  :manifest_props => "
-    as_override => true,
-  ",
+  :title_pattern  => "#{BgpLib::ASN} yellow 3.3.3.3 ipv4 unicast",
+  :remote_as      => 3,
```

Comment:
What was the reasoning behind this change?  Is '2 yellow 3.3.3.3 3' a valid value for remote_as on nexus?